aero-camera-streaming-daemon/aero-camera-streaming-daemon_1.0/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: camera-streaming-daemon
+Priority: optional
+Maintainer: Ritul Jasuja <ritul.jasuja@intel.com>
+Build-Depends: debhelper (>=9), dh-systemd (>= 1.5), dh-autoreconf, git, python-future, libavahi-client-dev, libavahi-core-dev, libavahi-glib-dev, libqt5gstreamer-1.0-0, libgstreamer-plugins-base1.0-dev, libgstrtspserver-1.0-dev
```

Comment:
libqt5gstreamer ?
Please check https://github.com/01org/camera-streaming-daemon/blob/master/configure.ac for actual list of dependencies.

Reply:
gstreamer-1.0 is just a list of dependency debian packages which includes libqt5gstreamer, gir1.2-gstreamer-1.0, and libqtgstreamer. To check if this package got installed, i have added one of the dependency package in the Build-Depends list.

Reply:
Why gstreamer-1.0 itself?

Reply:
ping

Reply:
As there is no package with the name gstreamer-1.0 everytime this check will fail. So one of the actual package that gets installed is added as dependency.